src/projections/analysis/SumDetailReader.java

```diff
@@ -165,7 +171,177 @@ protected void read()
 		buildTable(TOTAL_TIME);
 	    } else if (label.equals("EPCallTimePerInterval")) {
 		buildTable(NUM_MSGS);
-	    } else {
+	    } else if (label.equals("MsgSentCount")) {

```

Comment:
This is all unnecessarily duplicated, it should all be using the same method with a parameter for each different type of data. As commented above, this should all be read as `RLEBlock`s like execution and call time already are, so ideally this should also be read in `buildTable` just as those are.